bundle/edu.gemini.model.p1.submit/src/main/scala/edu/gemini/model/p1/submit/Post.scala

```diff
@@ -15,15 +15,16 @@ private[submit] object Post {
 
     for {
       pdf1 <- p.meta.firstAttachment.toRight(ClientError("The proposal is missing PDF attachment 1.")).right
-      pdf2 <- p.meta.secondAttachment.toRight(ClientError("The proposal is missing PDF attachment 2.")).right
-      _   <- doPost(con, p, pdf1, pdf2).right
+      pdf2 <- Right(p.meta.secondAttachment).right
```

Comment:
Do we still need the `.right` projection?  I guess we're using an old version of Scala.

Reply:
We do, this is scala-2.11